docs/src/main/asciidoc/security-openid-connect-web-authentication.adoc

```diff
@@ -468,6 +468,53 @@ quarkus.oidc.logout.post-logout-uri-param=returnTo
 quarkus.oidc.logout.extra-params.client_id=${quarkus.oidc.client-id}
 ----
 
+[[back-channel-logout]]
+==== Back-Channel Logout
+
+link:https://openid.net/specs/openid-connect-backchannel-1_0.html[Back-Channel Logout] is used by OpenId Connect providers to logout the current user from all the applications this user is currently logged in, bypassing the user agent.
+
+You can configure Quarkus to support `Back-Channel Logout` as follows:
+
+[source,properties]
+----
+quarkus.oidc.auth-server-url=http://localhost:8180/realms/quarkus
+quarkus.oidc.client-id=frontend
+quarkus.oidc.credentials.secret=secret
+quarkus.oidc.application-type=web-app
+
+quarkus.oidc.back-channel-logout.path=/back-channel-logout
```

Comment:
Wouldn't be better `quarkus.oidc.logout.backchannel.path` in order to reuse the same config category `logout`?

Reply:
@pedroigor thanks for the comments, but `quarkus.oidc.logout` is dedicated to RP initiated logout, this is why I did not want to make it a parent of `backchannel` related properties since these 2 logout mechanisms are orthogonal .
But I'll try and see how it looks